frontend/templates/product-list/sections/HeroSection.tsx

```diff
@@ -27,16 +27,18 @@ export function HeroSection({ productList }: HeroSectionProps) {
    return (
       <Flex direction="column">
          <HeroTitle>
-            {productList.title}
+            {productList?.overrides?.title}
             {page > 1 ? ` - Page ${page}` : ''}
          </HeroTitle>
-         {productList.tagline && (
+         {productList?.overrides?.tagline && (
```

Comment:
Do we ever use `productList.tagline` now? Is it only used in the fallback logic?

Reply:
yes